common/version.go

```diff
@@ -6,23 +6,39 @@ import (
 	pbcommon "github.com/drand/drand/protobuf/common"
 )
 
-const (
-	FallbackMajor = 0
-	FallbackMinor = 0
-	FallbackPatch = 0
+// Must be manually updated!
+// Before releasing: Verify the version number and set Prerelease to ""
+// After releasing: Increase the Patch number and set Prerelease to "-pre"
+var version = Version{
+	Major:      1,
+	Minor:      1,
+	Patch:      2,
+	Prerelease: "+pre",
+}
```

Comment:
This should be a `v1.4.0+pre`, no?
We are currently running the release v1.3.1 (see https://github.com/drand/drand/releases/)

Reply:
Thanks for that, I did not know where to look for the next version.